proposals/simd/BinarySIMD.md

```diff
@@ -28,177 +28,177 @@ The `v8x16.shuffle` instruction has 16 bytes after `simdop`.
-| `i32x4.trunc_sat_f32x4_u`  |    `0xac`| -                  |
-| `f32x4.convert_i32x4_s`    |    `0xaf`| -                  |
-| `f32x4.convert_i32x4_u`    |    `0xb0`| -                  |
-| `v8x16.swizzle`            |    `0xc0`| -                  |
-| `v8x16.shuffle`            |    `0xc1`| s:LaneIdx32[16]    |
-| `v8x16.load_splat`         |    `0xc2`| -                  |
-| `v16x8.load_splat`         |    `0xc3`| -                  |
-| `v32x4.load_splat`         |    `0xc4`| -                  |
-| `v64x2.load_splat`         |    `0xc5`| -                  |
-| `i8x16.narrow_i16x8_s`     |    `0xc6`| -                  |
-| `i8x16.narrow_i16x8_u`     |    `0xc7`| -                  |
-| `i16x8.narrow_i32x4_s`     |    `0xc8`| -                  |
-| `i16x8.narrow_i32x4_u`     |    `0xc9`| -                  |
-| `i16x8.widen_low_i8x16_s`  |    `0xca`| -                  |
-| `i16x8.widen_high_i8x16_s` |    `0xcb`| -                  |
-| `i16x8.widen_low_i8x16_u`  |    `0xcc`| -                  |
-| `i16x8.widen_high_i8x16_u` |    `0xcd`| -                  |
-| `i32x4.widen_low_i16x8_s`  |    `0xce`| -                  |
-| `i32x4.widen_high_i16x8_s` |    `0xcf`| -                  |
-| `i32x4.widen_low_i16x8_u`  |    `0xd0`| -                  |
-| `i32x4.widen_high_i16x8_u` |    `0xd1`| -                  |
-| `i16x8.load8x8_s`          |    `0xd2`| m:memarg           |
-| `i16x8.load8x8_u`          |    `0xd3`| m:memarg           |
-| `i32x4.load16x4_s`         |    `0xd4`| m:memarg           |
-| `i32x4.load16x4_u`         |    `0xd5`| m:memarg           |
-| `i64x2.load32x2_s`         |    `0xd6`| m:memarg           |
-| `i64x2.load32x2_u`         |    `0xd7`| m:memarg           |
-| `v128.andnot`              |    `0xd8`| -                  |
-| `i8x16.avgr_u`             |    `0xd9`|                    |
-| `i16x8.avgr_u`             |    `0xda`|                    |
-| `i8x16.abs`                |    `0xe1`| -                  |
-| `i16x8.abs`                |    `0xe2`| -                  |
-| `i32x4.abs`                |    `0xe3`| -                  |
+| `i16x8.load8x8_s`          |    `0x01`| m:memarg           |
+| `i16x8.load8x8_u`          |    `0x02`| m:memarg           |
+| `i32x4.load16x4_s`         |    `0x03`| m:memarg           |
+| `i32x4.load16x4_u`         |    `0x04`| m:memarg           |
+| `i64x2.load32x2_s`         |    `0x05`| m:memarg           |
+| `i64x2.load32x2_u`         |    `0x06`| m:memarg           |
+| `v8x16.load_splat`         |    `0x07`| m:memarg           |
+| `v16x8.load_splat`         |    `0x08`| m:memarg           |
+| `v32x4.load_splat`         |    `0x09`| m:memarg           |
+| `v64x2.load_splat`         |    `0x0a`| m:memarg           |
+| `v128.store`               |    `0x0b`| m:memarg           |
+| `v128.const`               |    `0x0c`| i:ImmByte[16]      |
+| `v8x16.swizzle`            |    `0x0d`| -                  |
```

Comment:
It looks like the shuffle/swizzle opcodes have swapped opcode numbers.

Reply:
Oops! Thanks for catching that :)